aviary/interface/methods_for_level2.py

```diff
@@ -73,18 +76,26 @@ def __init__(self, verbosity=None, **kwargs):
         self.verbosity = verbosity
         set_warning_format(verbosity)
 
-        self.model = AviaryGroup()
+        self.problem_type = problem_type
+        if problem_type == ProblemType.MULTI_MISSION:
+            self.model = om.Group()
+        else:
+            self.model = AviaryGroup()
+            self.aviary_inputs = None
+
+        self.aviary_groups_dict = {}
 
-        self.aviary_inputs = None
+        self.meta_data = BaseMetaData.copy()
 
     def load_inputs(
         self,
         aircraft_data,
         phase_info=None,
         engine_builders=None,
         problem_configurator=None,
-        meta_data=BaseMetaData,
+        meta_data=None,
         verbosity=None,
+        check=False,
```

Comment:
check should default to True on the user-facing load_inputs, so that the user doesn't ever have to type it. I think the case where they will ever need to call `check_and_preprocess_inputs` is pretty rare.

Reply:
I'm planning on leaving this as check=False because 1) we don't want to brick current users setups that are already working 2) the whole check system will be deprecated in the future as all users will be required to load .csv inputs with a separate function.